### PR TITLE
add part of HouseSDJpaService unit tests

### LINKED ISSUE #105: HouseSDJpaService unit tests
## Task description

There is a need to create unit tests for `HouseSDJpaService`.
As an example, you can have a look at `UserControllerTest`.

## Task Relationships
This task is a sub-task of: #100

### PR DESCRIPTION
**Pull request by DDmit04**
## 🚀 Description

Create tests for `HouseService` class

## 📄 Motivation and Context

Closes #105

### CODE REVIEW

**Review by jmprathab — APPROVED** [review]
Looks good.

File: src/test/java/com/myhome/services/unit/HouseSDJpaServiceTest.java [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyNoInteractions;
+
+class HouseSDJpaServiceTest {
+
+  private final int TEST_HOUSES_COUNT = 10;
```
Comment (mslowiak) [inline src/test/java/com/myhome/services/unit/HouseSDJpaServiceTest.java:37]: `private static final`

File: src/test/java/com/myhome/services/unit/HouseSDJpaServiceTest.java [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+        .willReturn(communityHouse);
+    given(houseMemberRepository.saveAll(membersToAdd))
+        .willReturn(membersToAdd);
+
+    // when
+    Set<HouseMember> resultMembers = houseSDJpaService.addHouseMembers(HOUSE_ID, membersToAdd);
+
+    // then
+    assertEquals(membersToAddSize, resultMembers.size());
+    assertEquals(membersToAddSize, communityHouse.getHouseMembers().size());
```
Comment (mslowiak) [inline src/test/java/com/myhome/services/unit/HouseSDJpaServiceTest.java:109]: This assertion is not needed - you are asserting the test data that you created with `getTestCommunityHouse` method

File: src/test/java/com/myhome/services/unit/HouseSDJpaServiceTest.java [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+    // then
+    assertFalse(isMemberDeleted);
+    verify(communityHouseRepository).findByHouseId(HOUSE_ID);
+    verify(communityHouseRepository, never()).save(communityHouse);
+    verifyNoInteractions(houseMemberRepository);
+  }
+
+  @Test
+  void getHouseDetailsById() {
+    // given
```
Comment (mslowiak) [inline src/test/java/com/myhome/services/unit/HouseSDJpaServiceTest.java:198]: Why is it empty?

File: src/test/java/com/myhome/services/unit/HouseSDJpaServiceTest.java [hunk 4]
```diff
@@ -0,0 +1,235 @@
...
+
+    // when
+
+    // then
+  }
+
+  @Test
+  void getHouseMembersById() {
+    // given
```
Comment (mslowiak) [inline src/test/java/com/myhome/services/unit/HouseSDJpaServiceTest.java:207]: Why is it empty?

**Comment by mslowiak** [comment]
@DDmit04 take a look on comments

**Pull request merged**